Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2759,6 +2759,20 @@ ExposeDefaultSpeakerAsSpecificDeviceEnabled:
     WebCore:
       default: true
 
+ExposeRollAngleAsTwistEnabled:
```

Comment:
This needs to be guarded with a HAVE/ENABLE/PLATFORM so it doesn’t show up on Mac.